docs/pages/access-controls/guides/moderated-sessions.mdx

```diff
@@ -105,7 +118,13 @@ The following are required options for `join_sessions`:
 |`kinds`|`[]`[Session kind](#session-kinds)|The kind of session that the policy applies to|
 |`modes`|`[]`[Participant mode](#participant-modes)|The participant mode that applies to the user joining the Moderated Session under this policy|
 
-#### Example
+<Admonition type="note">
+Moderator or observers can only join via CLI currently with `tsh join --mode=observer|moderator <session id>`.  Web joining is 
```

Comment:
I would rephrase this a bit.

> In order to join a session in `moderator` or `observer` mode, users must join
> using the CLI: `tsh join --mode=observer|moderator`.
> Joining from the web UI always joins in `participant` mode.

Reply:
Modified to mention k8s not available from UI as this is now available for Servers